main.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from rotary_jet_spinning import *
+
+
+cwd = os.getcwd()
+
+deck = Deck(cwd + "/" + "deck.yaml")
+
+machine = RJSMachine(deck)
+
+polymer = Polymer(deck)
+
+features = GraphFeatures(deck)
+
+model = RJSModel(polymer, machine)
+
+data = Data(deck, polymer, machine, model, features)
+
+organization = Organization(data, deck, machine, polymer, model, features)
+
+organized_data = organization.organize_data(data, deck, machine, polymer, model, features)
+
+graph = PointGraph(organized_data)
+
+# comparison = Comparison(deck, polymer, machine, model)
```

Comment:
Just to make sure, the comparison has not yet been implemented (or is not part of this PR)?

Reply:
It is not part of this PR. I am waiting for this one to be closed before doing a PR for the comparison with experiment part.